packages/terra-status-view/tests/wdio/status-view-spec.js

```diff
@@ -0,0 +1,158 @@
+/* global browser, Terra */
+
+const viewports = Terra.viewports('tiny', 'medium');
+
+describe('StatusView', () => {
+  describe('Displays a default status view', () => {
+    beforeEach(() => browser.url('/#/tests/status-view-tests/default'));
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-status-view-padding-bottom': '10px',
+      '--terra-status-view-padding-top': '10px',
+      '--terra-status-view-title-color': 'purple',
+      '--terra-status-view-title-font-family': 'Impact',
+      '--terra-status-view-title-font-size': '36px',
+    });
+  });
+
+  describe('Displays an error status view with all props given', () => {
+    beforeEach(() => browser.url('/#/tests/status-view-tests/error-all-props'));
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-status-view-padding-bottom': '10px',
```

Comment:
Do we need to continually test padding-bottom, padding-top, title-color, font-family, and font-size for each variation if they will always have the same values?

Reply:
I am doing it to test as many theming props as I can for each scenario, may be a bit overkill but wanted to be thorough